litmus-portal/graphql-server/manifests/subscriber.yml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
```

Comment:
remove the unnecessary changes, the yaml was in a minified form but your editor has formatted. Just keep the namespace changes and remove the other formatting changes.

Reply:
okay sure.